xarray/backends/api.py

```diff
@@ -222,6 +228,10 @@ def open_dataset(filename_or_obj, group=None, decode_cf=True,
     --------
     open_mfdataset
     """
+    
+    if mask_and_scale is None:
+        mask_and_scale = not engine == 'pseudonetcdf'
```

Comment:
Actually, let's do this for `decode_cf` instead of `mask_and_scale`. I don't think the other netCDF specific decoding like concatenating characters is relevant to psuedonetcdf, either.

Also, let's be sure to change this for `open_mfdataset` (and `open_dataarray`) as well.

Reply:
Next push:
* I switched from `mask_and_scale` to `decode_cf`.
* `open_mfdataset` uses a pass-through of keywords.
* `open_dataarray` has been updated

FYI - This leads to an odd outcome -- if I want to `decode_times`, I have to explicitly enable `decode_cf` and then disable all the others individually, *and* enable `decode_times`.

Reply:
Oh... I didn't realize you do want to decode times. In that case, I guess disabling only mask_and_scale is a reasonable choice.